README.md

```diff
@@ -728,79 +772,94 @@ environments should be exposed as an input variable. For example, the frontend-a
-   for `stage/frontend-app/terraform.tfvars`, you might have:
+For example, `stage/frontend-app.tfvars` may look like this:
    
-    ```hcl
-    instance_count = 3
-    instance_type = "t2.micro"
-    ```
+```hcl
+terragrunt = {
+  include {
+    path = "${find_in_parent_folders()}"
+  }
+}
 
-    Whereas for `prod/frontend-app/terraform.tfvars`, you might have:
-   
-    ```hcl
-    instance_count = 10
-    instance_type = "m2.large"
-    ```
-1. `.terragrunt`: This is the typical Terragrunt configuration file, which specifies how to manage remote state and 
-   locking. 
+instance_count = 3
+instance_type = "t2.micro"
+```
+
+And `prod/frontend-app.tfvars` may look like this:
    
-This gives you the following folder structure:   
+```hcl
+terragrunt = {
+  include {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+instance_count = 10
+instance_type = "m2.large"
+```
+
+This gives you the following file layout:   
  
 ```
 infrastructure-live
   └ stage
     └ frontend-app
-        └ .terragrunt
-        └ terraform.tfvars
+      └ terraform.tfvars
     └ backend-app
     └ mysql
     └ vpc
   └ prod
     └ frontend-app
-        └ .terragrunt
-        └ terraform.tfvars
+      └ terraform.tfvars
     └ backend-app
     └ mysql
     └ vpc
 ```
- 
-Notice how there are no Terraform configurations (`.tf` files) in any of the folders. To tell Terragrunt where to
-find your Terraform code, each `.terragrunt` file can contain a `terraform` block:
+
+Notice how there are no Terraform configurations (`.tf` files) in any of the folders. Instead, each `.tfvars` file 
+tells Terragrunt where to find your Terraform code using a `terraform` block:
 
 ```hcl
-terraform {
-  source = "git::git@github.com:foo/bar.git//frontend-app?ref=v0.0.1"
+terragrunt = {
+  include {
+    path = "${find_in_parent_folders()}"
+  }
+  
+  terraform {
+    source = "git::git@github.com:foo/bar.git//frontend-app?ref=v0.0.1"
+  }
 }
 ```
 
 *(Note: the double slash (`//`) is intentional and required. It's part of Terraform's Git syntax for [module 
 sources](https://www.terraform.io/docs/modules/sources.html).)
 
-When you run Terragrunt and it finds a `terraform` block in your configuration, Terragrunt will:
+When you run Terragrunt and it finds a `terraform` block, it will:
  
 1. Download the configurations specified via the `source` parameter into a temporary folder. This is done using the
    [terraform init command](https://www.terraform.io/docs/commands/init.html), so the `source` parameter supports the
    exact same syntax as the [module source](https://www.terraform.io/docs/modules/sources.html) parameter, including
    local file paths, Git URLs, and Git URLs with `ref` parameters (useful for checking out a specific tag, commit, or
    branch of Git repo). Terragrunt will download all the code in the repo (i.e. the part before the double-slash `//`) 
```

Comment:
1. Tried to clarify it a bit here: https://github.com/gruntwork-io/terragrunt/pull/122/commits/6c9f37be58a90d3b3515c85697d5b45d2f754ab2
1. It looks backwards compatible.